src/js/utils/__tests__/ServiceUtil-test.js

```diff
@@ -322,9 +322,9 @@ describe('ServiceUtil', function () {
         it('adds the networkName field to the service', function () {
           let service = ServiceUtil.createServiceFromFormModel({
             containerSettings: { image: 'redis' },
-            networking: { networkType: 'user', ports: [{}] },
+            networking: { networkType: 'prod', ports: [{}] },
```

Comment:
I've to admit it's a little confusing and the prop name is anything but good (that's a different story and not part of this PR)... The network type is used to define whether service container network should be `HOST`, `BRIDGE`, or `USER` but it is also used to define the service `ipAddress.networkName`. Thus we assign the selected option to the `networkType` property and use that prop to determine the respective values. That's why we expect `service.ipAddress.networkName` to equal `networkType` and the reason why I adjusted the values.